Lib/xml/dom/minidom.py

```diff
@@ -1904,7 +1921,8 @@ def renameNode(self, n, namespaceURI, name):
 
 def _clone_node(node, deep, newOwnerDocument):
     """
-    Clone a node and give it the new owner document.
+    Returns a copy of node.
```

Comment:
Remove \n after """.  Add after this line.  But this revision seems wrong as it omits mention of newOwnerDocument.  I would have to read code to suggest improvement.